docs/release-notes/self-hosted/self-hosted-v4.4.0.md

```diff
@@ -0,0 +1,55 @@
+---
+rss_title: Codacy release notes RSS feed
+rss_href: /feed_rss_created.xml
+description: Release notes for Codacy Self-hosted v4.4.0.
+codacy_tools_version_old: https://github.com/codacy/codacy-tools/releases/tag/3.8.3
+codacy_tools_version_new: https://github.com/codacy/codacy-tools/releases/tag/3.9.16
+---
+
+# Self-hosted v4.4.0
+
+These release notes are for [Codacy Self-hosted v4.4.0](https://github.com/codacy/chart/releases/tag/4.4.0){: target="_blank"}, released on October 12, 2021. <!-- TODO Update release date -->
+
+To upgrade Codacy, follow [these instructions](../../chart/maintenance/upgrade.md).
+
+📢 [Visit the Codacy roadmap](https://roadmap.codacy.com) and <span class="skip-vale">let us know</span> your feedback on both new and planned product updates!
+
+<!--TODO Check these issues
+
+Jira issues without release notes
+
+Epics:
+-   https://codacy.atlassian.net/browse/CY-4617
+-   https://codacy.atlassian.net/browse/CY-4408
+-   https://codacy.atlassian.net/browse/DOCS-209
+-   https://codacy.atlassian.net/browse/DOCS-42
```

Comment:
These are actually two Epics that I use as "buckets" of related tasks, and since I made a few commits when working on a couple of tasks, they showed up here. We can safely ignore them. :)